src/distrs/gamma.jl

```diff
@@ -22,3 +25,80 @@ function gammalogpdf(k::T, θ::T, x::T) where {T<:Real}
     val = -loggamma(k) + xlogy(k - 1, xθ) - log(θ) - xθ
     return x < 0 ? oftype(val, -Inf) : val
 end
+
+function gammacdf(k::T, θ::T, x::T) where {T<:Real}
+    # Handle the degenerate case
+    if iszero(k)
+        return last(promote(k, θ, x, x >= 0))/sqrt(one(θ))
```

Comment:
Similarly, what's the reason for this specific heuristic and should this maybe be handled in SpecialFunctions?

Reply:
Same reasoning as above